invenio_oaiserver/response.py

```diff
@@ -100,57 +104,68 @@ def identify(**kwargs):
     e_tree, e_identify = verb(**kwargs)
 
     e_repositoryName = SubElement(
-        e_identify, etree.QName(NS_OAIPMH, 'repositoryName'))
+        e_identify, etree.QName(NS_OAIPMH, 'repositoryName')
+    )
     e_repositoryName.text = cfg['OAISERVER_REPOSITORY_NAME']
 
     e_baseURL = SubElement(e_identify, etree.QName(NS_OAIPMH, 'baseURL'))
     e_baseURL.text = url_for('invenio_oaiserver.response', _external=True)
 
-    e_protocolVersion = SubElement(e_identify,
-                                   etree.QName(NS_OAIPMH, 'protocolVersion'))
+    e_protocolVersion = SubElement(
+        e_identify, etree.QName(NS_OAIPMH, 'protocolVersion')
+    )
     e_protocolVersion.text = cfg['OAISERVER_PROTOCOL_VERSION']
 
     for adminEmail in cfg['OAISERVER_ADMIN_EMAILS']:
         e = SubElement(e_identify, etree.QName(NS_OAIPMH, 'adminEmail'))
         e.text = adminEmail
 
     e_earliestDatestamp = SubElement(
-        e_identify, etree.QName(
-            NS_OAIPMH, 'earliestDatestamp'))
+        e_identify, etree.QName(NS_OAIPMH, 'earliestDatestamp')
+    )
     earliest_date = datetime(MINYEAR, 1, 1)
-    earliest_record = OAIServerSearch(
-        index=current_app.config['OAISERVER_RECORD_INDEX']).sort({
-            "_created": {"order": "asc"}})[0:1].execute()
+    earliest_record = (
+        current_oaiserver.search(index=current_app.config['OAISERVER_RECORD_INDEX'])
+        .sort({"_created": {"order": "asc"}})[0:1]
```

Comment:
```suggestion
        .sort({current_oaiserver.created_key: {"order": "asc"}})[0:1]
```

Just saw this one... We need another config like `OAISERVER_CREATED_KEY` (so that we can override it in RDM to be `"updated"`). I've updated #198 accordingly.